app.py

```diff
@@ -13,8 +12,8 @@ def post_random_game() -> Union[str, Response]:
     args = request.args
     user = args.get('user')
     try:
-        return jsonify(dataclasses.asdict(get_random_game(user))
-    except UserIsNoneError as exc:
+        return jsonify(dataclasses.asdict(get_random_game(user)))
+    except Exception as exc:
```

Comment:
I would have a separate `Exception` block _after_ the `UserIsNone` error, as that would indicate that there is a separate error than a 404 "User Not Found" deal. This would probably be a 500 status code in my opinion.

The reason you want a separate exception block is that different exceptions should be handled differently (i.e. a `UserNotFound` exception will return a 404 status code while any other exception would get a 500 status code.) Also, if and when you add logging you can log at different levels for each one (for example, log an `info` that a user isn't found [which is normal behavior for bad input] vs. log an `exception` when any other exception is raised [because that would be exceptional!])